src/scripts/main.js

```diff
@@ -1,7 +1,29 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const element = document.createElement('div');
+  const body = document.querySelector('body');
+  const heading = document.createElement('h2');
+  const descr = document.createElement('p');
+
+  body.append(element);
+
+  element.style.right = posRight + 'px';
+  element.style.top = posTop + 'px';
+  element.style.display = 'none';
+
+  element.className = `${type} + notification`;
+
+  heading.textContent = title;
+  heading.className = 'title';
+  element.insertAdjacentElement('afterbegin', heading);
+
+  descr.textContent = description;
+  heading.insertAdjacentElement('afterend', descr);
+
+  setTimeout(function() {
+    element.style.display = 'block';
+  }, 2000);
```

Comment:
Almost done)
First append and then remove after 2 sec 
![image](https://user-images.githubusercontent.com/70009205/212132108-7b410ed8-4b34-420f-a0f6-8778f7806b0e.png)